Host/Startup.cs

```diff
@@ -22,13 +24,13 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.AddMvcCore().AddJsonFormatters().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.AddSingleton<MongoDbContext>(new MongoDbContext("mongodb://localhost"));
+            services.AddTransient<ImageRepository>();
```

Comment:
We can use MongoDbContext with this repository. Or you would like to use repository?

Reply:
For now I will not change it due to netural changes.